### PR TITLE
Dowload test fixes

### PR DESCRIPTION
**Pull request by marusak**
I have no idea how it worked before...

### CODE REVIEW

File: src/ImageDeleteModal.jsx [hunk 1]
```diff
@@ -5,13 +5,21 @@ import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
+function sortTags(a, b) {
+    if (a.endsWith(":latest"))
+        return -1;
+    if (b.endsWith(":latest"))
+        return 1;
+    return a > b;
```
Comment (martinpitt) [inline src/ImageDeleteModal.jsx:13]: This looks wrong. The compare function is supposed to return -1, 0, or 1, not false/true -- turning the latter into a number in Python just  yields NaN. How about `a.localeCompare(b)`?

File: src/ImageDeleteModal.jsx [hunk 2]
```diff
@@ -5,13 +5,21 @@ import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
+function sortTags(a, b) {
+    if (a.endsWith(":latest"))
```
Comment (martinpitt) [inline src/ImageDeleteModal.jsx:9]: so `RepoTags` really contains the `:` in the names? (That sounds strange)

**Comment by martinpitt** [comment]
I applied the first two commits (test fixes) to my local Fedora gating VM, and they do fix the test. :balloon: thanks!

File: src/ImageDeleteModal.jsx (hunk 2 above)
Comment (marusak) [reply src/ImageDeleteModal.jsx:9]: repo tags are the full strings with registry, name and tag.
`[ "docker.io/library/busybox:latest", "docker.io/library/busybox:1", "docker.io/library/busybox:2", "docker.io/library/busybox:3" ]` for example

File: src/ImageDeleteModal.jsx (hunk 1 above)
Comment (marusak) [reply src/ImageDeleteModal.jsx]: fixed using `localCompare`

**Pull request merged**